docs/guides/getting-started.md

```diff
@@ -0,0 +1,140 @@
+# Getting Started
+
+You can create a `react-sketchapp` project with `skpm`, by cloning a ready-made [example](../examples.md), or by manually setting up the `package.json` and `manifest.json` scripts (advanced usage).
+
+## Environment Setup
+
+You will need npm, Node and Sketch.
+
+- Code editor e.g. [VSCode](https://code.visualstudio.com/), [Atom](https://atom.io/)
+- Node.js & `npm` – [install with Homebrew](https://nodejs.org/en/download/package-manager/#macos) (or install with [Node Version Manager](https://nodejs.org/en/download/package-manager/#nvm))
+- [Sketch](https://www.sketch.com/)
+  - requires macOS
+
+## Creating a Project With Skpm
+
+**Replace** `my-app` with your desired project name:
+
+### Installation
+
+```bash
+npm i -g skpm
```

Comment:
👍 I like to avoid global installs where possible, would be nice to have alternative docs, but if someone cares about that, chances are they know how to do `npm install --save skpm` and `npx skpm`, or copy the `package.json`, etc manually. I agree it's better to be explicit that it's a global install.